### ISSUE TITLE
license missing

### ISSUE THREAD

**Issue by PabloLION** [AriSweedler/Typing-Test#1]
Hi Ari, I want to reuse your code for a new website. May I have permission? I see no license for this project so I think it's free-to-use. Please reply / PM me in case if you disagree. Thanks in advance.

**Comment by AriSweedler** [comment]
Hi, what part do you want to use? I have learned a lot since writing this, so I might be able to suggest a better piece of code for you to look at, as my old code is full of errors. Use at your own risk!

That being said, please feel free to do whatever you want with the code in this repo.

**Comment by PabloLION** [comment]
I cloned this repo and make a reversive change haha. I like your simplest UI.

**Comment by AriSweedler** [comment]
That's super cool. I'm totally looking forward to seeing what you create haha :)

**Comment by PabloLION** [comment]
Oh I built this app [allingo](https://pablion.github.io/allingo/) a year ago, here's the [repo](https://github.com/Pablion/allingo)
Going to rewrite it with react in 5 months or so...

**Comment by AriSweedler** [comment]
That is amazing! You've created something seriously awesome!!! Thanks for following through :)

**Comment by AriSweedler** [comment]
I am especially interested in how you got the IPA to show up, but I could not read your code in the repo. Did you minify it? Or generate it with a framework? I would expect the source code to be in the repo, and the build artifacts generated when you want to deploy your site, but... Ah well. You've built something that's looks and is totally awesome

**Comment by PabloLION** [comment]
Thanks for the compliment. I built it with vue obviously, and AWS. \
I forgot how I deployed it, but I just find it for you: I just git push the `dist` folder. 

```zsh
    cdallin    # this is my customized zsh command
    set -e
    npm run build # build
    cd dist       # navigate into the build output directory
    git init
    git add -A
    git commit -m 'deploy'
    git push -f git@github.com:Pablion/allingo.git master:gh-pages
    # deploy to https://Pablion.github.io/allingo
```

When I built it (on my early days), I wanted to make it private. If you want I can send the source code to to your email... As I said I'm rewriting it with react and more functionality, so I'm not going to maintain that one.

**Comment by PabloLION** [comment]
By the way the IPA is from an dictionary API

**Comment by AriSweedler** [comment]
I'd love to read the source, yes. I will not share your code with anyone, just read it myself for fun. My email is ari@sweedler.com

**Comment by PabloLION** [comment]
@AriSweedler I email you the package without `node_modules`. You can delete the message with your email for privacy(since you didn't put it in your profile page).
And by the way, 
1. The site `https://ari.sweedler.com/` gives me a 400 error. 
2. Your CA seems not working (HTTPS not unsecure).

**Comment by PabloLION** [comment]
Hi again, Ari.
I just retrieved my files from iCloud on my Mac... It was so hard with windows.

I made [some gist](https://gist.github.com/Pablion/6e2cf2d31e6e4b86abfe92b7611c372e) for the voice and the dictionary API. 
Sorry that I can't send the whole source code because some secret keys like API keys was hard coded into the file, and I don't have time to check all the files. They were not ready to share, since I was not going to publish them.

for the TTS, I used amazon Polly: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Polly/Presigner.html#getSynthesizeSpeechUrl-property
fot the IPA I used this API:
https://rapidapi.com/dpventures/api/wordsapi
